README.md

```diff
@@ -55,15 +57,19 @@ and extension not works in `producation` mode.
 1. Install the Chrome [Rax Developer Tools](https://chrome.google.com/webstore/detail/rax-developer-tools/anpeoinhjjligmgoiepbnigjhmijblff) extension
 2. Reload and go to the 'Rax' tab in the browser's development tools
 
-### VS Code Extensions
+#### VS Code Extensions
+
+You can use [Rax VS Code Extensions](https://github.com/raxjs/rax-vscode-extensions) to get better development experience.
+
+### Awesome Things 📝
 
-You can also use the [Rax VS Code Extensions](https://github.com/raxjs/rax-vscode-extensions) to get better development experience.
+You can find some awesome things in the [awesome-rax](https://github.com/raxjs/awesome-rax).
```

Comment:
You can find some awesome resources in [awesome-rax](https://github.com/raxjs/awesome-rax).